### PR TITLE
Remove duplicated call to action and smooth routing

### LINKED ISSUE #752: Remove Login button on signed out homepage
Two calls to action can confuse some people. Just "Login and begin filing" on the button in the body would help here.

### PR DESCRIPTION
**Pull request by wpears**
Closes #752

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
...
-    "js:dev": "rimraf ./dist/js/app.min.js.map && browserify -d -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -e ./src/js/index.js -o ./dist/js/app.min.js",
+    "js": "rimraf ./dist/js/app.min.js.map && rimraf ./dist/js/app.min.js.bak && browserify -d -g [ envify --NODE_ENV ${NODE_ENV:-production} ] -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -p [ minifyify --map app.min.js.map --output ./dist/js/app.min.js.map --uglify [ -cm ] ] ./src/js/index.js | bundle-collapser - > ./dist/js/app.min.js",
+    "js:dev": "rimraf ./dist/js/app.min.js.map && rimraf ./dist/js/app.min.js.bak && browserify -d -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -e ./src/js/index.js -o ./dist/js/app.min.js",
```
Comment (awolfe76) [inline package.json]: 🚀  had noticed this one too!

Comment (awolfe76) [reply package.json]: Actually, `js:dev` still doesn't build right. You'll see things like `##KEYCLOAK_URL##/` in the console after running `js:dev`. I think _maybe_ it needs `env` run too?

**Review by awolfe76 — CHANGES_REQUESTED** [review]
Just the one thing on the build stuff. Other than that 👍

File: package.json (hunk 1 above)
Comment (wpears) [reply package.json]: Fixed!

**Pull request merged**